lib/option.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, script_path: str) -> None:
         self.headers = {}
         if option.headers:
             for h in option.headers:
-                hn, hv = h.split(':')
+                hn, hv = h.split(': ')
```

Comment:
可以加上num参数修复：
```suggestion
                hn, hv = h.split(':', 1)
```
感觉这样比较通用，不会因为冒号后面少打个空格就报错